apps/remix-ide-e2e/src/tests/ballot.test.ts

```diff
@@ -92,6 +92,25 @@ module.exports = {
         })
   },
 
+  'Compile with remappings set in remappings.txt file #group1': function (browser: NightwatchBrowser) {
+    browser
+      .clickLaunchIcon('filePanel')
+      .click('*[data-id="workspacesMenuDropdown"]')
+      .click('*[data-id="workspacecreate"]')
+      .waitForElementVisible('*[data-id="modalDialogCustomPromptTextCreate"]')
+      .waitForElementVisible('[data-id="fileSystemModalDialogModalFooter-react"] > button')
+      // eslint-disable-next-line dot-notation
+      .execute(function () { document.querySelector('*[data-id="modalDialogCustomPromptTextCreate"]')['value'] = 'workspace_remix_default' })
+      .waitForElementPresent('[data-id="fileSystemModalDialogModalFooter-react"] .modal-ok')
+      .execute(function () { (document.querySelector('[data-id="fileSystemModalDialogModalFooter-react"] .modal-ok') as HTMLElement).click() })
+      .pause(1000)
+      .waitForElementVisible('*[data-id="treeViewLitreeViewItemcontracts"]')
+      .addFile('contracts/lib/storage/src/Storage.sol', { content: retrieverContract })
+      .addFile('remappings.txt', { content: 'storage=contracts/lib/storage/src' })
+      .addFile('contracts/Retreiver.sol', { content: storageContract })
+      .verifyContracts(['Retreiver', 'Storage'])
```

Comment:
typo, the test should actually fails.

Reply:
no, remappings are defined, so it will pass.

Reply:
I mean you are checking `Retreiver` but the contract is named `Retriever`

Reply:
Ah, fixed it.